### ISSUE TITLE
3.0.0 bar always shown even when no message is currently selected

### ISSUE THREAD

**Issue by xyzzy-xyzzy** [MailHops/mailhops-plugin#11]
Subject says it all. 

I don't know if this has anything to do with it but I also use the [Manually Sort Folders](https://addons.mozilla.org/en-us/thunderbird/addon/manually-sort-folders/?src=cb-dl-popular) addon.  It has an setting to "start with the following folder open" which I set to my Inbox.  So at startup the inbox is chosen but no messages are selected to be displayed.  Thus nothing is shown in the message panel in 2.0.0 as expected.  

Now with 3.0.0 it shows the mailhops bar.  Indeed since I have the startup this way, when I installed 3.0.0 and Thunderbird restarted, there was that blue highlighted mailhops bar conspicuously showing in the otherwise previously empty pane containing the red preferences icon, a hops popup menu that of course has nothing to popup, a refresh button with nothing to refresh, and a "unsubscribe" link "button".

Bottom line, the bar shouldn't be shown if there is no message to accompany it.  In 2.0.0 the mailhops info was "part" of a message's header (which I prefer without the extraneous highlighting, see issue #10) so this was never an issue.
